content/docs/command-reference/remote/modify.md

```diff
@@ -375,24 +389,24 @@ authentication method:
   $ dvc remote modify --local myremote client_secret 'client-secret'
   ```
 
-- `account_name` - storage account name. May work by itself (via
-  [anonymous auth](https://docs.microsoft.com/en-us/azure/storage/blobs/anonymous-read-access-configure))
-  or along with either `account_key` or `sas_token` along with this):
+* `account_key` - storage account key:
 
   ```dvc
-  $ dvc remote modify --local myremote account_name 'myuser'
+  $ dvc remote modify --local myremote account_key 'mykey'
   ```
 
-* `account_key` - storage account key (for `account_name`):
+* `sas_token` - shared access signature token:
 
   ```dvc
-  $ dvc remote modify --local myremote account_key 'mysecret'
+  $ dvc remote modify --local myremote sas_token 'mytoken'
   ```
 
-* `sas_token` - shared access signature token (for `account_name`):
+* `allow_anonymous_login` - whether fallback to anonymous login or not. If none
+  of the options above are specified beside `account_name`, and option is set
+  then DVC will try to connect anonymously for read-only buckets:
```

Comment:
```suggestion
* `allow_anonymous_login` - whether to fallback to anonymous login. If none
  of the options above are specified besides `account_name`, DVC will try
  to connect anonymously for read-only buckets:
```

Reply:
Nice. Can probably simplify even more? E.g. "Only has an effect along with `account_name`."

Reply:
read-only -> public?

Reply:
> read-only -> public?

yes

Reply:
> read-only -> public

updated comment.

> Only has an effect along with

doesn't have the same meaning as the (admittedly clunky) phrase

> none of the options above are specified besides